MMA.md

```diff
@@ -0,0 +1,365 @@
+    "Telemetry Opt In" -> true          // If the user has not disabled Sending Anonymous Product Usage.
+  }"
+  "appId" -> "app_6Ao...."              // Leanplum App ID.
+  "clientKey" -> "dev_srwDUNZR...."     // Leanplum client access key.
+  "systemName" -> "iOS"                 // Fixed String in SDK.
+  "locale" -> "zh_TW"                   // System Locale.
+  "timezone" -> "Asia/Taipei"           // System Timezone.
+  "versionName" -> "55.0a1"             // Fennec version.
+  "systemVersion" -> "10.3.1"            // System version.
+  "deviceModel" -> "iPhone"             // System device model.
+  "timezoneOffsetSeconds" -> "28800"    // User timezone offset with PST.
+  "deviceName" -> "sdaswani-31710"       // System device name.
+  "region" -> "(detect)"                // Not used. We strip location so this is will be the default stub value in Leanplum SDK.
+  "city" -> "(detect)"                  // Same as above.
+  "country" -> "(detect)"               // Same as above.
+  "location" -> "(detect)"              // Same as above.
+  "newsfeedMessages" -> " size = 0"     // Not used. New Leanplum Inbox message(Leanplum feature) count.
+  "includeDefaults" -> "false"          // Not used. Always false.
+
+  // Other life cycle actions
+  "action" -> "heartbeat"               // heartbeat: every 15 minutes when app is in the foreground
+                                        // pauseSession: when app goes to background
+                                        // resumeSession: when app goes to foreground
+
+  // Sent for every action
+  "userId" -> "b13b3c239d01aa7c"        // Set by Fennec, we use random uuid so users are anonymous to Leanplum.
+  "deviceId" -> "b13b3c239d01aa7c"      // Same as above.
+  "sdkVersion" -> "2.2.2-SNAPSHOT"      // Leanplum SDK version.
+  "devMode" -> "true"                   // If the SDK is in developer mode. For official builds, it's false.
+  "time" -> "1.497595093902E9"          // System time in second.
+  "token" -> "nksZ5pa0R5iegC7wj...."    // Token come from Leanplum backend.
+
+~~~~~~~~~~~~~~~
+
+Notes on what data is collected
+-------------------------------
+
+User Identifier:
+Since Device ID is a random UUID, Leanplum can't map the device to any know Client ID in Fennec nor Advertising ID.
+
+Events:
+Most of the Leanplum events can be mapped to a single combination of Telemetry event (Event+Method+Extra).
+Some events are not collected in Mozilla Telemetry. This will be addressed separately in each campaign review.
+There are three elements that are used for each event. They are: event name, value(default: 0.0), and info(default: "").
+Default value for event value is 0.0. Default value for event info is empty string.
+
+Here is the list of current Events sent, which can be found here in the code base: https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Application/LeanplumIntegration.swift#L21
```

Comment:
Great, thank you so much for this link! That definitely makes it clear where these probes actually live in the code. I'd actually just link to the filename itself (rather than the specific line).